docs/source/configuration/refine_admin.rst

```diff
@@ -0,0 +1,35 @@
+Refine Admin
+============
+
+Some administrative functionality is built out using `Refine <https://refine.dev>`_. This is a separate service and requires a bit of setup to make it work with the MITxOnline app. 
+
+Accessing
+---------
+
+The Docker compose environment builds a ``refine`` container that runs a dev server available at port 8016. 
+
+Set Up
+------
+
+The Refine app uses OAuth2/OIDC for authentication with the core MITxOnline app. You will need to generate a shared private key for this authentication flow to work. 
+
+*Generating the Key:* The following command will generate the key:
```

Comment:
A note about the location of file where it will be placed after running the command. Know it will be current path but good to add for more understanding.